docs/FAQ.md

```diff
@@ -0,0 +1,94 @@
+# Frequently asked questions
+
+### Why does my branch not move to the new commit after `jj new/commit`?
+
+If you're familiar with Git, you might expect the current branch to move forward
+when you commit. However Jujutsu does not have a concept of a "current branch".
+
+To move branches, use `jj branch set`. 
+
+### I made a commit and `jj git push --all` says "Nothing changed" instead of pushing it. What do I do?
+
+`jj git push --all` pushes all _branches_, not all revisions. You have two 
+options: 
+
+* Using `jj git push --change` will automatically create a branch and push it.
+* Using `jj branch` commands to create or move a branch to either the commit 
+you want to push or a descendant on it. Unlike Git, Jujutsu doesn't do this 
+automatically (see previous question).
+
+### Where is my commit, why is it not visible in `jj log`? 
+
+Is your commit visible with `jj log -r 'all()'`? 
+
+If yes, you should be aware that `jj log` only shows the revisions matching 
+`revsets.log` by default. You can change it as described in [config] to show 
+more revisions. 
+
+If not, the revision may have been abandoned (e.g because you used `jj abandon`
+, or because it's an obsolete version that's been rewritten with `jj rebase`, 
+`jj describe`, etc) In that case, `jj log -r commit_id` should show the 
+revision as "hidden". `jj new commit_id` or `jj edit commit_id` should make the
+revision visible again.
+
+See [revsets] and [templates] for further guidance.
+
+### How do I add a file? 
+
+Jujutsu automatically commits new files to the current working-copy commit.
+If you think the current state of the working-copy commit is fine, use `jj new`.
+Then future edits will go into a new working-copy commit on top of the now 
+former working-copy commit. Whenever your are happy with another set of edits,
+use `jj squash` to amend the previous commit. 
+
+For more options see the next question.
+
+### How can I emulate workflows like `git add -p` or `hg commit -i`?
```

Comment:
I think the question should be more descriptive, something like:

> ### Can I add a portion of the edits I made to a file to a commit, similarly to `git add -p` or `hg commit -i`.

I'd also remove the note about "In the future...". Instead, it'd be nice if the PR adding that option also added that command here.

Reply:
done, rephrased to "We may add ..."

Reply:
I still think it'd be better to just remove that paragraph, but I'll leave it up to you.

Reply:
removed